docs/diagram.qdbd

```diff
@@ -0,0 +1,47 @@
+# Go to https://app.quickdatabasediagrams.com/#/
```

Comment:
It looks like this change is already in the main branch, we should rebase/remove this commit from the PR.

Reply:
Yes it is! Forgot to rebase after initializing, thanks!